tests/test_rbuser_tostring.py

```diff
@@ -0,0 +1,80 @@
+"""RedBrick Test Module; Tests the to_string method of the rbuser module."""
+
+import unittest
+from useradm import rbuser
+
+
+class RBUserTestCase(unittest.TestCase):
+    """Test Case class for to_string method"""
+
+    def setUp(self):
+        """Call before every test case."""
+        self.billy_attr = {'cn': 'Billy Bo',
```

Comment:
using testuser1 makes more sense for tests. its better for tests for searching and reviewing the tests as they grow and you test with multiple users with different attributes 
and clling them billy and bob gives us no benefits